src/components/ha-selector/ha-selector-select.ts

```diff
@@ -182,6 +187,13 @@ export class HaSelectSelector extends LitElement {
     `;
   }
 
+  private _getTranslation(option, label) {
+    if (!this.selector.select?.translation_key) {
+      return label;
+    }
+    return option; // TODO translate option
```

Comment:
This part needs to get the fetched localization, that is fetched at the option-flow dialog.

Reply:
The full key will be : `component.${domain}.selector.${translation_key}.options.${option}`.
The main problem is that we don't have the domain in this component.

Reply:
I could bring in the `domain` by extending the selector, but a `computeOptionLabel` scenario sounds even better, but what should I do to implement this, as it will an be optional feature. Further, I tries to fetch the translations, but they do not come up, is `this.hass.localize()` globalized?